src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea, Colorable {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. Here, `FigureArea` and `Colorable` may have different responsibilities, so they should not be implemented together if they represent different concepts.